doc/devel/documenting_mpl.rst

```diff
@@ -43,16 +43,17 @@ guide, developers guide, api reference, and FAQs. The documentation suite is
 built as a single document in order to make the most effective use of cross
 referencing.
 
-.. note::
-
-   An exception to this are the directories :file:`examples` and
-   :file:`tutorials`, which exist in the root directory.  These contain Python
-   files that are built by `Sphinx Gallery`_.  When the docs are built,
-   the directories :file:`docs/gallery` and :file:`docs/tutorials`
-   are automatically generated. Do not edit the rst files in :file:docs/gallery
-   and :file:docs/tutorials, as they are rebuilt from the original sources in
-   the root directory.
-
+Sphinx_ also creates ``.rst`` files that are staged in :file:`doc/api` from
+the docstrings of the classes in the Matplotlib library.  Except for
+:file:`doc/api/api_changes/`, these ``.rst`` are created when the
+documentation is built.
+
+Similarly, the contents of :file:`docs/gallery` and :file:`docs/tutorials` are
+generated by the `Sphinx Gallery`_ from the sources in :file:`examples` and
```

Comment:
not directly related but I think the "examples" folder should be renamed "gallery" or the "gallery" folder on the website should be renamed "examples"...

Reply:
Yeah I agree.  Its very confusing.  I prefer "examples".